test/parallel/test-tls-check-server-identity.js

```diff
@@ -1,16 +1,16 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var util = require('util');
+const common = require('../common');
+const assert = require('assert');
```

Comment:
@koxauvin Please don't remove the `hasCrypto` check. It is needed for this test. But please move the assignments to `assert` and `util` to be after that block.